docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       context: .
       target: dev
       args:
-        PHP_VERSION: 8.0
+        PHP_VERSION: "8.2"
```

Comment:
V `composer.json` mame `^8.2`.

Jinak puvodni `8.0` se ukazalo celkem tricky, protoze bez uvozovek to YAML naparsuje jako float a prevede na prostou `8`, takze v `Dockerfile` jako `php:8-cli` a zacalo to instalovat PHP 8.5 😄